packages/docs/src/helpers/CodeSandboxExample.ts

```diff
@@ -37,31 +41,30 @@ const getCodeSandboxHtml = ({ requireIcons = false }: CodesandboxConfig): string
     >
     ${requireIcons ? iconsStyles : ''}
     <div id="app"></div>
+    <script type="module" src="/src/main.js"></script>
   `
 }
 
 const getCodeSandboxMain = ({ requireIcons = false }: CodesandboxConfig): string => {
-  if (requireIcons) {
-    return iconsConfig
-  }
+  if (requireIcons) { return iconsConfig }
 
   return main
 }
 
 const packageJson = ({ dependencies = {}, devDependencies = {} }: CodesandboxConfig): string => {
   const commonDeps = {
-    'core-js': '^3.6.5',
     vue: '^3.0.0',
     'vuestic-ui': `${packageUi.version}`,
   }
   const commonDevDeps = {
-    '@vue/cli-plugin-babel': '~4.5.0',
-    '@vue/cli-service': '~4.5.0',
-    '@vue/compiler-sfc': '^3.0.0',
+    '@vitejs/plugin-vue': '^3.0.3',
+    vite: '^3.0.7',
```

Comment:
@m0ksem, @asvae do I need to implement the auto picking the version from the `vuestic-ui/package.json`? Or better replace it with a `package-name@latest`?

Reply:
Latest will be good. We will now that something is broken :)